vignettes/tidytranscriptomics.Rmd

```diff
@@ -375,19 +390,15 @@ An alternative to MDS for examining relationships between samples is using hiera
 counts_scal_MDS %>% 
 	
 	# extract 500 most variable genes
-	keep_variable(
-	              .sample = sample, 
-	              .transcript = transcript, 
-	              .abundance = count_scaled, 
-	              top = 500) %>%
+	keep_variable( .abundance = count_scaled, top = 500) %>%
 		
 	# create heatmap
 	heatmap(
 	      .column = sample,
 	      .row = transcript,
 	      .value = count_scaled,
 	      annotation = c(dex, cell),
-	      transform = log1p
+	      transform = function(x) x %>% log1p %>% tidyHeatmap:::scale_robust()
```

Comment:
```suggestion
	      transform = log1p
```
So I think if you accept this suggested change to revert this PR should hopefully pass